inc/meta-box-html.php

```diff
@@ -178,6 +178,10 @@ private function draw_input( $field, $form_id = NULL ) {
 			HTML::text_area( $field['rows'], $field['cols'], $field['meta_key'], $value, $title, $label, $field['placeholder'], $required, $form_id );
 		}
 
+		if ( $field['type'] == 'wysiwyg' ) {
+			$this->wysiwyg( $value, $field['meta_key'], $field['params'], $label, $form_id );
+		}
```

Comment:
All the other fields types here take `$title` as an argument, should `wysiwyg` and use it, if it has a value, in the label in line 252?